.gitignore

```diff
@@ -22,7 +22,7 @@ nbactions.xml
 *.ipr
 *.iws
 .idea/
-local/
+local
```

Comment:
Why change this? If i recall the slash indicates you want to ignore the directory, you've changed it to ignoring a single file.

Reply:
The trailing slash means it *has* to be a directory. This allows it to be either. In my new setup, I have the one master local directory and all the other roots have a link to that directory. This allows for both.

Reply:
Fair enough